internal/core/testing.go

```diff
@@ -329,7 +329,7 @@ func Test(config *TestConfig) func(t *testing.T) {
 		executeCmd := func(args []string) interface{} {
 			stdoutBuffer := &bytes.Buffer{}
 			stderrBuffer := &bytes.Buffer{}
-			logger.Debugf("command: %s", args)
+			log.Debugf("command: %s", args)
```

Comment:
```suggestion
			log.Debugf("executing command: %s", args)
```

Reply:
Wondering of all the `executing command`  should be put in the bootstrap function 🤔

Reply:
It would make sense for me to see only command, before and afterfunc execution. Why are execute commands required here?